src/controllers/accommodation.js

```diff
@@ -2,11 +2,10 @@ import { config } from 'dotenv';
 import models from '../db/models';
 
 config();
-
 export default class accomodationFacility {
   static async getAllAccommodations(req, res) {
     try {
-      const accommodations = await models.Accommodation.findAll({ where: { userId: req.user.id } });
+      const accommodations = await models.Accommodation.findAll();
```

Comment:
can you explain the reason you changed the findAll conditions?

Reply:
@kagabof  the reason is, all the users should be able to to view available accommodation. before it was only seen by the person who have created them either the manager or the supplier but considering that Users will need to view those accommodation to rate them or any other reason so I felt that it should be seen by every authenticated user

Reply:
Okay.